examples/ml-gke.yaml

```diff
@@ -40,19 +40,32 @@ deployment_groups:
         - range_name: services
           ip_cidr_range: 10.0.32.0/20
 
+  - id: gke_service_account
+    source: community/modules/project/service-account
+    settings:
+      name: gke-service-account
+      project_roles:
+      - logging.logWriter
```

Comment:
Would we not need more permissions like compute.createResources, networking permissions, etc?

Reply:
AFAIK, that's not required. We're passing the service to the GKE cluster (and GKE node pool), which uses this SA in the node_config argument. This is the SA the nodes will use. [parameter definition](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#service_account). I'm running all GKE integration tests to see if there's something I've not caught

Reply:
Ack, thanks!